app/src/main/java/de/tum/in/tumcampusapp/activities/MainActivity.java

```diff
@@ -118,6 +125,21 @@ public void onDrawerOpened(View drawerView) {
         } else {
             initAdapter();
         }
+
+        tapCounter = 0;
+        findViewById(R.id.main_toolbar).setOnClickListener(view -> {
+            tapCounter++;
```

Comment:
That `tapCounter` incremented on every tap, even after quite some time.
I feel like only tapping the logo in quick succession should toggle the logo. I'd suggest tp remember the last time of tap and reset the `tapCounter` to 1 when more than one second has passed after the last tap

Reply:
its a bit overkill. The start screen is only visible for a few seconds.